phone-book/src/app.js

```diff
@@ -0,0 +1,177 @@
+class App {
+  constructor(appContainerId) {
+    this.state = {};
```

Comment:
it's better to initialize properties with null value for example.
It would tell about your app more.

To understand logic of your app I have to read all logic, instead of just understanding have you been designed state correctly or not